maxim/tests/test_anthropic.py

```
@@ -1,128 +1,222 @@
-import json
-import logging
+import dotenv
 import os
 import unittest
+from unittest.mock import patch
```

Ruff (0.11.9)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)